docs/reference/metadata.md

```diff
@@ -0,0 +1,188 @@
+# Metadata reference
+
+**Summary**
+
+The following document is a reference guide for the internal metadata schema for bibliographic records in InvenioRDM.
+
+**Intended audience**
+
+The guide is intended for advanced users, administrators and developers of InvenioRDM with significant prior experience.
+
+## Overview
+
+InvenioRDM's bibliographic records are stored as JSON documents in a structure
+that is aligned with DataCite's Metadata Schema v4.x with minor additions
+and modifications.
+
+**Schema version**
+
+All records contain a schema definition in the top-level key ``$schema``. The value
+is a link to the internal JSONSchema which is being used to validate the structure of the record.
+The field is fully system-managed.
+
+```json
+{
+    "$schema": "https://localhost/schemas/records/record-v1.0.0.json",
+    ...
+}
+```
+
+**Top-level fields**
+
+Following is an overview of the top-level fields in a record:
+
+- ``id``/``pid`` - The internal persistent identifier for a specific version.
```

Comment:
Just wondering if tables are easier to read instead of bullet points. For me tables help a lot on indentation/next line, visually speaking.

Reply:
Yeah, I agree. Just very cumbersome to do tables in Markdown :-)

Reply:
Done on the last commit :)